platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/ClearTrigger.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.wiring;
+
+/**
+ * A placeholder object, to provide a type to the input `clear` wire for wiring components.
+ */
+public record ClearTrigger() {}
```

Comment:
Are we planning to have multiple control commands like this where we create records, whether empty or with parameters? If we have multiple commands common across multiple components, we might want a generic InputWire where the Command Interface defines all the command records that are usable with it.

I'm just thinking about how we would generalize this pattern.

Reply:
There will definitely be other instances of this pattern. I've observed at least one other case while working on PCES, also containing no parameters.

I can't think of any benefits off the top of my head from making a common command interface, but I'm not opposed to the idea if there's a clean way that makes things easier to understand.